src/core/dagpair.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { expect } from "chai";
+import { MemoryDatastore } from "interface-datastore";
+import { isBrowser } from "browser-or-node";
+import { Buffer } from "buffer";
+import { Peer, BlockStore } from "..";
+import { setupLibP2PHost } from "../setup";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const writeKey = require("libp2p/src/pnet").generate;
+
+const swarmKey = Buffer.alloc(95);
+writeKey(swarmKey);
+
+const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms));
+
+let p1: Peer;
+let p2: Peer;
+
+// Don't run these tests in the browser, they'd require a relay
+(isBrowser ? describe.skip : describe)(
```

Comment:
Don't run this test in the browser for now... otherwise we'd need to add a relay node or something.